### PR TITLE
Boost 1.3.0 Beta

### PR DESCRIPTION
**Pull request by thingalon**
This PR updates the changelogs for Jetpack Boost and the Jetpack Connection package to support Jetpack Boost 1.3.0 beta.

Please review the changelogs and make sure they make sense.

#### Changes proposed in this Pull Request:
- Updates changelogs in preparation for beta release.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
Read the changelogs, make sure they make sense.

### CODE REVIEW

File: projects/plugins/boost/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,26 @@ All notable changes to this project will be documented in this file.
...
+- Critical CSS: More descriptive error message if critical css is failing because of x-frame-options deny config.
+- Speed Scores: Added "without Boost" speed score indicator.
+
+### Changed
+- Critical CSS: Take port numbers into account when comparing origins for proxying.
+
+### Fixed
+- Critical CSS: Clear generated CSS on theme change.
+- Critical CSS: Ensure generator process is resumed after module deactivated and reactivated without reload.
+- Speed Scores: Clear speed score on plugin deactivation.
```
Comment (karthikax) [inline projects/plugins/boost/CHANGELOG.md:26]: and uninstall?

**Review by karthikax — DISMISSED** [review]

File: projects/plugins/boost/CHANGELOG.md (hunk 1 above)
Comment (thingalon) [reply projects/plugins/boost/CHANGELOG.md:26]: Is it done on deactivation, uninstall or both?

Comment (karthikax) [reply projects/plugins/boost/CHANGELOG.md:26]: On both steps.
Ref: https://github.com/Automattic/jetpack/blob/master/projects/plugins/boost/app/class-jetpack-boost.php#L161
https://github.com/Automattic/jetpack/blob/master/projects/plugins/boost/app/modules/critical-css/class-critical-css.php#L166

**Review by karthikax — APPROVED** [review]

**Comment by thingalon** [comment]
Adding @kraftbj and @jeherve as reviewers since this touches on non-boost files. :)

**Review by jeherve — APPROVED** [review]
LGTM!

**Pull request merged**